test-configs.yaml

```diff
@@ -751,19 +735,55 @@ device_types:
       - whitelist: {defconfig: ['defconfig']}
       - blacklist: {kernel: ['v3.']}
 
-  qemu_arm:
+  qemu_arm-gicv2:
```

Comment:
I prefer to qualify better the machine, it is a virt-gicv2

Reply:
Sure.